<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemViagemWidget</class>
 <widget class="QWidget" name="ItemViagemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblNome">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Nome da Viagem</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblDestino">
     <property name="text">
      <string>Destino:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblData">
     <property name="text">
      <string>Data:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnExcluir">
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditar">
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
